frontend/src/component/common/SegmentItem/SegmentItem.tsx

```diff
@@ -110,7 +114,11 @@ export const SegmentItem: FC<SegmentItemProps> = ({
                 disableGutters
                 TransitionProps={{ mountOnEnter: true, unmountOnExit: true }}
             >
-                <StyledAccordionSummary id={`segment-accordion-${segment.id}`}>
+                <StyledAccordionSummary
+                    id={`segment-accordion-${segment.id}`}
+                    tabIndex={-1}
+                    aria-controls={segmentDetailsId}
```

Comment:
The [a11y recommendations for accordion](https://v5.mui.com/material-ui/react-accordion/#accessibility) say to use id and aria-controls. If you set aria-controls, MUI will set that as the id of the details panel. In our case, the accordion summary doesn't do anything, but because the id is now set on the details, we can use that to indicate that our preview button now controls the section.

I thiiink this is a sensible thing to do.